dev-packages/node-integration-tests/suites/fs-instrumentation/.gitignore

```diff
@@ -0,0 +1,11 @@
+some-file.txt.copy
```

Comment:
Can we just make this: `some-file.txt.*` ?

Reply:
and maybe move them out of the root of the package, into e.g. `fixtures/` or something like this?